SurveyApp/Modules/Login/LoginViewController.swift

```diff
@@ -33,15 +43,127 @@ final class LoginViewController: UIViewController {
+        }
+
+        emailField.snp.makeConstraints {
+            $0.top.equalTo(logoImageView.snp.bottom).offset(120.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+
+        passwordField.snp.makeConstraints {
+            $0.top.equalTo(emailField.snp.bottom).offset(20.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+
+        forgotButton.snp.makeConstraints {
+            $0.top.equalTo(emailField.snp.bottom).offset(20.0)
+            $0.trailing.equalTo(passwordField.snp.trailing)
+            $0.leading.equalToSuperview().inset(280.0)
+            $0.height.equalTo(55.0)
+        }
+
+        loginButton.snp.makeConstraints {
+            $0.top.equalTo(passwordField.snp.bottom).offset(20.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+    }
+    
     private func setUpViews() {
-        view.backgroundColor = .white
+        let backgroundImage = R.image.loginBackground()
+        let logoImage = R.image.logo()
+
+        navigationController?.isNavigationBarHidden = true
+
+        backgroundImageView.image = backgroundImage
+
+        logoImageView.image = logoImage
+        logoImageView.contentMode = .scaleAspectFit
+
+        forgotButton.backgroundColor = .clear
+        forgotButton.setTitle("Forgot?", for: .normal)
+        forgotButton.titleLabel?.font = .regular(ofSize: .body)
+        forgotButton.tintColor = UIColor.white.withAlphaComponent(0.3)
+
+        loginButton.backgroundColor = .white
+        loginButton.setTitle("Log in", for: .normal)
+        loginButton.titleLabel?.font = .bold(ofSize: .body)
+        loginButton.tintColor = .black
+        loginButton.layer.cornerRadius = 10.0
+        
+        setUpTextField()
+        setUpBlurOverlayView()
+    }
+    
+    private func setUpTextField() {
+        emailField.placeholder = "Email"
+        emailField.setPlaceholderColor(color: UIColor.white.withAlphaComponent(0.3))
+        emailField.layer.cornerRadius = 12.0
+        emailField.setPadding(left: 10.0, right: 10.0)
+        emailField.textColor = .white
+        emailField.backgroundColor = UIColor.white.withAlphaComponent(0.18)
+        emailField.keyboardType = .emailAddress
+        emailField.autocorrectionType = .no
+        emailField.autocapitalizationType = .none
+        emailField.font = .regular(ofSize: .body)
+
+        passwordField.placeholder = "Password"
+        passwordField.setPlaceholderColor(color: UIColor.white.withAlphaComponent(0.3))
+        passwordField.layer.cornerRadius = 12.0
+        passwordField.setPadding(left: 10.0, right: 10.0)
+        passwordField.textColor = .white
+        passwordField.backgroundColor = UIColor.white.withAlphaComponent(0.18)
+        passwordField.isSecureTextEntry = true
+        passwordField.font = .regular(ofSize: .body)
```

Comment:
Both of these textfields have exactly the same UI. so maybe instead of creating extension for setPlaceholderColor() you can instead create a UITextfield subclass that initialises with the correct properties and then the only thing you have to set is the placeholder text